manifest.json

```diff
@@ -3157,6 +3386,614 @@
+        },
+        "me.badhaloninja.InspectorExpandAll": {
+            "name": "InspectorExpandAll",
+            "description": "Adds a button to the inspector to expand all visible slots on an inspector",
+            "category": "Inspectors",
+            "sourceLocation": "https://github.com/badhaloninja/InspectorExpandAll",
+            "authors": {
+                "badhaloninja": {
+                    "url": "https://github.com/badhaloninja"
+                }
+            },
+            "versions": {
+                "1.0.0": {
+                    "releaseUrl": "https://github.com/badhaloninja/InspectorExpandAll/releases/tag/v1.0.0",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/badhaloninja/InspectorExpandAll/releases/download/v1.0.0/InspectorExpandAll.dll",
+                            "filename": "InspectorExpandAll.dll",
+                            "sha256": "a8deefcefb252abcf2baac7046293cb9b30aae533c66c8de4166af8a58ce61ba"
+                        }
+                    ] 
+                }
+            }
+        },
+        "me.badhaloninja.CreateNewAdditions": {
+            "name": "CreateNewAdditions",
+            "description": "Adds more primitives to the CreateNewDialog",
+            "category": "Technical Tweaks",
+            "sourceLocation": "https://github.com/badhaloninja/CreateNewAdditions",
+            "authors": {
+                "badhaloninja": {
+                    "url": "https://github.com/badhaloninja"
+                }
+            },
+            "versions": {
+                "1.0.0": {
+                    "releaseUrl": "https://github.com/badhaloninja/CreateNewAdditions/releases/tag/v1.0.0",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/badhaloninja/CreateNewAdditions/releases/download/v1.0.0/CreateNewAdditions.dll",
+                            "filename": "CreateNewAdditions.dll",
+                            "sha256": "9de58b2afb452f7f881b8035c0abc80c5ab676f0a34b2ee3f060b55b3d47a5b4"
+                        }
+                    ] 
+                }
+            }
+        },
+        "me.New-Project-Final-Final-WIP.HeadlessTweaks": {
+            "name": "HeadlessTweaks",
+            "description": "Adds some nice to have features to headless clients",
+            "category": "Misc",
+            "sourceLocation": "https://github.com/New-Project-Final-Final-WIP/HeadlessTweaks",
+            "authors": {
+                "badhaloninja": {
+                    "url": "https://github.com/badhaloninja"
+                },
+                "Spex": {
+                    "url": "https://github.com/PointerOffset"
+                },
+                "New-Project-Final-Final-WIP": {
+                    "url": "https://github.com/New-Project-Final-Final-WIP"
+                }
+            },
+            "versions": {
+                "1.2.0": {
```

Comment:
Didn't review properly, but I think we can just trust it enough to not be malicious, since didn't see anything clearly out of place.